cider/exec/nextgen/jitlib/llvmjit/LLVMJITFunction.cpp

```diff
@@ -263,6 +264,45 @@ JITValuePointer LLVMJITFunction::getArgument(size_t index) {
   }
 }
 
+JITValuePointer LLVMJITFunction::packJITValuesImpl(
```

Comment:
This function is used for transfer ```vector``` to ```array```?

Reply:
I'm afraid not, this function is just used to pack Join keys for call the loopUpbyKey function